javascript/mcp-server/README.md

```diff
@@ -31,3 +33,11 @@ Add the following to `claude_desktop_config.json`. On Mac, you can find this in
   }
 }
 ```
+
+Or you can run
+
+```bash
+npx forevervm-mcp install --claude
+```
+
+to automate the setup step.
```

Comment:
let's put this section first since it's the preferred setup. We could even remove the manual setup from the readme here and just include it on a docs page (once we have a docs site set up I imagine we will have a page for MCP)